src/warnings/message.js

```diff
@@ -5,6 +5,6 @@ export default {
     browserManipulationsOnRemoteBrowser:     'The screenshot and window resize functionalities are not supported in a remote browser. They can function only if the browser is running on the same machine and in the same environment as the TestCafe server.',
     screenshotNotSupportedByBrowserProvider: 'The screenshot functionality is not supported by the "{providerName}" browser provider.',
     resizeNotSupportedByBrowserProvider:     'The window resize functionality is not supported by the "{providerName}" browser provider.',
-    maximizeNotSupportedByBrowserProvider:   'The maximize functionality is not supported by the "{providerName}" browser provider.',
+    maximizeNotSupportedByBrowserProvider:   'The window maximization functionality is not supported by the "{providerName}" browser provider.',
```

Comment:
Oh, this could break tests. We need to run tests without `[docs]` tag if we want land this change with this PR.